src/index.html

```diff
@@ -11,12 +11,55 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo-image"
+          class="logo-img"
+        />
+      </a>
+      <nav class="nav-menu">
+        <ul class="menu-list">
+          <li class="link"><a class="nav-link is-active">Apple</a></li>
+          <li class="link"><a class="nav-link">Samsung</a></li>
+          <li class="link"><a class="nav-link">Smartphones</a></li>
+          <li class="link">
+            <a
+              class="nav-link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="link"><a class="nav-link">Gadgets</a></li>
+          <li class="link"><a class="nav-link">Tablets</a></li>
+          <li class="link"><a class="nav-link">Photo</a></li>
+          <li class="link"><a class="nav-link">Video</a></li>
```

Comment:
The anchor tags within the navigation menu do not have 'href' attributes. This will prevent them from functioning as links. Consider adding appropriate 'href' attributes to these anchor tags to ensure they navigate to the correct pages.